src/components/sidebar/components/sidebarDesktop/SidebarDesktopLineItemHistory.tsx

```diff
@@ -82,30 +84,42 @@ function HistoryItem({
   isLastChat,
 }: HistoryItemProps) {
   const [isHovered, setIsHovered] = useState(false)
+  const { mutate } = useUpdateChat()
+  const handleChange = (value: string) => {
+    mutate({ id: chatId, title: value })
+  }
 
   return (
-    <div
-      className={cn('flex justify-between gap-2')}
+    <Link
       onMouseEnter={() => setIsHovered(true)}
       onMouseLeave={() => setIsHovered(false)}
+      href={`/p/${postId}/c/${chatId}`}
+      className={cn('flex w-full justify-between gap-2 text-left')}
```

Comment:
tiny detail - the `cn` would not be needed here

Reply:
true